RecoLocalCalo/HGCalRecProducers/test/testHGCalRecoLocal_cfg.py

```diff
@@ -11,8 +11,8 @@
 process.load('FWCore.MessageService.MessageLogger_cfi')
 process.load('Configuration.EventContent.EventContent_cff')
 process.load('SimGeneral.MixingModule.mixNoPU_cfi')
-process.load('Configuration.Geometry.GeometryExtended2023LRecoReco_cff')
-process.load('Configuration.Geometry.GeometryExtended2023LReco_cff')
+process.load('Configuration.Geometry.GeometryExtended2023simReco_cff')
```

Comment:
@davidlange6 - A while ago, we decided that since sim and LReco geometries were identical, we should only keep one. We ended up keeping sim, even though it's being used with the LReco workflow right now.